sdk/src/wallet/wallet/builder.rs

```diff
@@ -56,11 +56,11 @@ impl<S: SecretManage> Default for WalletBuilder<S> {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[cfg(feature = "storage")]
 #[cfg_attr(docsrs, doc(cfg(feature = "storage")))]
-pub(crate) struct StorageOptions {
-    pub(crate) storage_path: PathBuf,
-    pub(crate) storage_file_name: Option<String>,
-    pub(crate) storage_encryption_key: Option<[u8; 32]>,
-    pub(crate) manager_store: ManagerStorage,
+pub struct StorageOptions {
```

Comment:
Should probably also get serde camelCase and then requires a migration?

Reply:
Yeah I was thinking about it, I also don't like the fields names

Reply:
Done